src/classes/AFFL_AccRecordType_TEST.cls

```diff
@@ -212,4 +212,78 @@ public with sharing class AFFL_AccRecordType_TEST {
         contact = [select Primary_Organization__c from Contact where ID=:contact.Id];
         System.assertEquals(acc.ID, contact.Primary_Organization__c);
     }
+
+    @isTest
+    public static void updateAdminAccRecordTypeBulk() {
```

Comment:
Yes, it is for #518. I don't think we need to create a test method for each bug if the bugs are related issue. In this case, the issue #518 and #524 are not related issues. We put it into the same PR is because they touch the same function.

The #518 occurs when we update more than 1 account's record type to the same record type (no contact and affiliation records need to be involved). So, I use a small batch (10 records) to verify it.

Maybe I should add comments above the method?

Reply:
Well I like the idea of more unit test methods. I am only a little concerned that test will run too long. Not a big question for this pull request but maybe we can figure out what worth creating a unit test.

Reply:
That is a very good point! 
Sometimes I feel like the test case can help me to make sure this scenario works , but it does increase the total execution time.